geometry_msgs/Vector3[] plan
voronoi/LineSegment[] roadmap